modules/python/python-dest.c

```diff
@@ -325,6 +331,21 @@ _inject_worker_insert_result_consts(PythonDestDriver *self)
   _inject_worker_insert_result(self, MAX);
 };
 
+static PyObject *
+py_get_persist_name(PythonDestDriver *self)
+{
+  PythonPersistMembers options =
+  {
+    .generate_persist_name_method = self->py.generate_persist_name,
+    .options = self->options,
+    .class = self->class,
+    .id = self->super.super.super.id
+  };
+
+  const gchar *persist_name = python_format_persist_name((LogPipe *)self, "python", &options);
```

Comment:
Consider replacing this call with the class inner method: `python_dd_format_persist_name`

Reply:
Added a refactor commit to the top. Thanks.